libraries/cyclestreets-view/src/main/java/net/cyclestreets/views/overlay/FindPlaceOverlay.kt

```diff
@@ -0,0 +1,66 @@
+package net.cyclestreets.views.overlay
+
+import android.content.Context
+import android.content.res.Resources
+import android.graphics.Canvas
+import android.graphics.Point
+import android.graphics.Rect
+import android.graphics.drawable.Drawable
+import androidx.core.content.res.ResourcesCompat
+import net.cyclestreets.Undoable
+import net.cyclestreets.view.R
+import net.cyclestreets.views.CycleMapView
+import org.osmdroid.api.IGeoPoint
+import org.osmdroid.api.IProjection
+import org.osmdroid.views.MapView
+import org.osmdroid.views.overlay.Overlay
+
+class FindPlaceOverlay(context: Context, cycleMapView: CycleMapView) : Overlay(), Undoable {
+    // (Passing CMV as parm so centreOn can be accessed, which we need to calc screen coords)
+    private val placeMarker_: Drawable?
+    private var cMapView_: CycleMapView
+    private var halfWidth: Int = 0
+    private var halfHeight: Int = 0
+
+    private val controller = OverlayHelper(cycleMapView).controller()
+
+    init {
+        val res: Resources = context.getResources()
+        placeMarker_ = ResourcesCompat.getDrawable(res, R.drawable.x_marks_spot, null)
+        if (placeMarker_ != null) {
```

Comment:
Do we really need this `if` protection?

Elsewhere we make Drawables from static resources be non-nullable (i.e. type of `Drawable` instead of `Drawable?`) and force non-nullability at the time of instantiation.  e.g. in `RotateMapOverlay.kt`, we do like:
```kt
private val onIcon: Drawable

init {
    onIcon = ResourcesCompat.getDrawable(context.resources, R.drawable.compass, null)!!
}
```

I think it would make the code simpler to take this approach here as well.